engines/director/score.cpp

```diff
@@ -842,6 +896,23 @@ bool Score::processImmediateFrameScript(Common::String s, int id) {
 	return false;
 }
 
+void Score::loadLingoNames(Common::SeekableSubReadStreamEndian &stream) {
+	if (_vm->getVersion() == 4) {
```

Comment:
Same question, is it only D4 or D4+? and function below.

Reply:
Fixed